playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -230,6 +230,14 @@
     - reconcile_scc_result.rc == 0
     run_once: true
 
+  - name: Migrate storage post policy reconcile
```

Comment:
This looks to be a PLAY that needs to be outdented from the preceding TASK phase.  See Travis errors.